components/feature/downloads/src/test/java/mozilla/components/feature/downloads/DownloadNotificationTest.kt

```diff
@@ -157,4 +158,28 @@ class DownloadNotificationTest {
         val summary = DownloadNotification.getSummaryList(testContext, listOf(download1, download2))
         assertEquals(listOf("mozilla.txt 10%", "mozilla2.txt 20%"), summary)
     }
+
+    @Test
+    fun getNotificationAccentColor() {
+        val download = DownloadJobState(
+            job = null,
+            state = DownloadState(fileName = "mozilla.txt", url = "mozilla.org/mozilla.txt", contentLength = 100L,
+                currentBytesCopied = 10,
+                status = DownloadState.Status.DOWNLOADING),
+            foregroundServiceId = 1,
+            downloadDeleted = false,
+            currentBytesCopied = 10,
+            status = DownloadState.Status.DOWNLOADING
+        )
+
+        val notification = DownloadNotification.createOngoingDownloadNotification(
+            testContext,
+            download,
+            notificationAccentColor = AbstractFetchDownloadService.Style().notificationAccentColor
+        )
+
+        val accentColor = ContextCompat.getColor(testContext, R.color.mozac_feature_downloads_notification)
```

Comment:
nit: Could we use `style.notificationAccentColor` as  `R.color.mozac_feature_downloads_notification` is an implementation detail and `style.notificationAccentColor` is the public api that we want to test :)

Reply:
You could create a variable for convenience: 

```kotlin
val style = AbstractFetchDownloadService.Style()
val notification = DownloadNotification.createOngoingDownloadNotification(
            testContext,
            download,
            notificationAccentColor = style.notificationAccentColor
        )

        val accentColor = ContextCompat.getColor(testContext, style)
```